src/app/pages/common/entity/entity-form/models/field-config.interface.ts

```diff
@@ -98,6 +91,7 @@ export interface FormComboboxOption {
 
 export interface FormDictConfig<P = any> extends BaseFieldConfig<P> {
   label?: string;
+  relation?: RelationGroup[];
```

Comment:
Isn't relation a base field though? `createFormGroup` in `EntityFormService` previously didn't care about the type of field config, when dealing with relations.
I suggest you test previous code to see if it supported relations on any type of field.

Reply:
From what I saw, the only things using those are the list, selectionlist and dict types which is why I moved it out of the base config.

Reply:
Please verify it by creating a simple form configuration somewhere with text inputs and relations.

Reply:
I put relation back into baseconfig and removed RelationConfig